app/models/vim_performance_analysis.rb

```diff
@@ -283,9 +281,6 @@ def get_vm_needs
       if VimPerformanceAnalysis.needs_perf_data?(options[:vm_options])
         # Get VM performance data
         start_time, end_time = get_time_range(options[:range])
-        # Set :only_cols for VM daily requested cols for better performance
-        options[:ext_options] ||= {}
```

Comment:
A. `get_daily_perf` will initialize this value if nil or {} is passed in.
B. `VimPerformanceAnalysis.get_daily_perf` basically just calls `VimPerformanceDaily.find_entries`. That method ignores `[:ext_options][:only_cols]`. So I was thinking that setting or passing `:only_cols` was pointless.